medusa/providers/generic_provider.py

```diff
@@ -480,6 +482,31 @@ def search(self, search_params, age=0, ep_obj=None):
         """Search the provider."""
         return []
 
+    @staticmethod
+    def parse_pubdate(pubdate, human_time=False, tz=None):
```

Comment:
I'd like to keep it in providers for now, as I don't really see any use outside of it in the near future.